rules/date-and-time-of-change-as-tooltip/rule.md

```diff
@@ -0,0 +1,30 @@
+---
+type: rule
+title: Dates – Do you provide the date and time of change as a tooltip?
+uri: date-and-time-of-change-as-tooltip
+authors:
+  - title: Ozair Ashfaque
+    url: https://www.ssw.com.au/people/ozair-ashfaque
+  - title: Tiago Araujo
+    url: https://www.ssw.com.au/people/tiago-araujo
+  - title: Joseph Fernandez
+    url: https://www.ssw.com.au/people/joseph-fernandez
+related: []
+created: 2023-03-15T03:01:02.285Z
+guid: 59ae02c2-961c-4f55-a874-97c56d8f9622
+---
+Displaying the date and time of change as a tooltip when users hover over the time of change can be an effective approach for interfaces with limited space or when providing both pieces of information together could lead to confusion. Tooltips allow users to access additional information about the context of the date and time of change without cluttering the main interface.
+
+<!--endintro-->
+
+::: bad
+![Figure: Bad Example – Cannot find date or time in tooltip on hover](bad-example-for-no-date-on-tooltip-while-hovering.png)
+:::
+
+::: good
+
+![Figure: Good Example – On hover, Facebook shows the date and time](bad-example-for-no-date-on-tooltip-while-hovering.png)
```

Comment:
Hey @ozairashfaqueSSW 

The image for the Facebook example is missing... here it shows a repetition of the bad example image...

1. Please upload the image 
2. Fix the image path accordingly